src/Raven.Server/Documents/Handlers/Processors/Subscriptions/AbstractSubscriptionsHandlerProcessorForUpdateSubscription.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System.Net;
+using System.Threading.Tasks;
+using JetBrains.Annotations;
+using Raven.Client.Documents.Subscriptions;
+using Raven.Client.Exceptions.Documents.Subscriptions;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Handlers.Processors.Subscriptions
+{
+    internal abstract class AbstractSubscriptionsHandlerProcessorForUpdateSubscription<TRequestHandler, TOperationContext> : AbstractSubscriptionsHandlerProcessorForCreateOrUpdateSubscription<TRequestHandler, TOperationContext>
+        where TOperationContext : JsonOperationContext
+        where TRequestHandler : AbstractDatabaseRequestHandler<TOperationContext>
+    {
+        protected AbstractSubscriptionsHandlerProcessorForUpdateSubscription([NotNull] TRequestHandler requestHandler) : base(requestHandler)
+        {
+        }
+
+        protected override bool GetDisabled(SubscriptionCreationOptions options) => false;
```

Comment:
why is it always `false`?

Reply:
because in [AbstractSubscriptionsHandlerProcessorForPostSubscription](https://github.com/ravendb/ravendb/blob/ca2bbebffa249b553c20ec9d054c9014f83642b0/src/Raven.Server/Documents/Handlers/Processors/Subscriptions/AbstractSubscriptionsHandlerProcessorForPostSubscription.cs#L12 ) you always call `CreateSubscriptionInternalAsync` with `disabled: false`